crates/sui-core/src/authority.rs

```diff
@@ -1965,7 +1965,6 @@ impl AuthorityState {
     }
 
     // This function is only used for testing.
-    #[cfg(test)]
     pub fn get_sui_system_state_object_for_testing(&self) -> SuiResult<SuiSystemState> {
```

Comment:
Should we change the name as well, if this is not only available in the `test` build?

Reply:
Looking at where this is used, should it be `#[cfg(any(test, msim))]`?